### PR TITLE
updated codeowners file based on ownership

### PR DESCRIPTION
**Pull request by mgaravin**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,28 @@
+# Shared
 *       @nupurjai @ahalimx86 @vishala-r @Eoghan1232
+Makefile       @nupurjai @ahalimx86 @vishala-r @Eoghan1232 @ahalimx86 @lpawlikx @ipatrykx
+go.mod       @nupurjai @ahalimx86 @vishala-r @Eoghan1232 @ahalimx86 @lpawlikx @ipatrykx
+go.sum       @nupurjai @ahalimx86 @vishala-r @Eoghan1232 @ahalimx86 @lpawlikx @ipatrykx
+README.md       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx
+proto/*       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx
+docs/*       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx
+.github/workflows/*       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx @mgaravin
```
Comment (ahalimx86) [inline .github/CODEOWNERS]: Can you please append @Eoghan1232 in the "Shared" and "Cloud Native" section please?

Comment (mgaravin) [reply .github/CODEOWNERS]: updated

**Review by Eoghan1232 — APPROVED** [review]

**Review by ahalimx86 — APPROVED** [review]

**Review by nupurjai — COMMENTED** [review]
Can we please add venkateshpnv & dhruvpt as well to our side of code

**Review by nupurjai — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,27 @@
...
 *       @nupurjai @ahalimx86 @vishala-r @Eoghan1232
+Makefile       @nupurjai @ahalimx86 @vishala-r @Eoghan1232 @ahalimx86 @lpawlikx @ipatrykx
+go.mod       @nupurjai @ahalimx86 @vishala-r @Eoghan1232 @ahalimx86 @lpawlikx @ipatrykx
+go.sum       @nupurjai @ahalimx86 @vishala-r @Eoghan1232 @ahalimx86 @lpawlikx @ipatrykx
+README.md       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx @Eoghan1232
+proto/*       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx @Eoghan1232
+docs/*       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx @Eoghan1232
+.github/workflows/*       @nupurjai @vishala-r @ahalimx86 @lpawlikx @ipatrykx @mgaravin @Eoghan1232
+
+# Cloud Native
```
Comment (paulfhncw) [inline .github/CODEOWNERS:11]: please add @paulfhncw to the code owners for cloud native files

**Review by paulfhncw — CHANGES_REQUESTED** [review]

**Pull request merged**